### PR TITLE
Implement DPT 251.600 for RGBW LED Controllers

### PR DESCRIPTION
**Pull request by fraxinas**
This adds a Data Point Type according to KNX Spec 251.600 which is used for RGBW LED Controllers.

### CODE REVIEW

File: src/objectcontroller.cpp [hunk 1]
```diff
@@ -2338,6 +2340,74 @@ void RGBObject::doSend(bool isWrite)
     Services::instance()->getKnxConnection()->write(getGad(), builder.getBuffer(), 5);
 }
 
+#ifdef STL_STREAM_SUPPORT_INT64
+RGBWObjectValue::RGBWObjectValue(const std::string& value)
```
Comment (2franix) [inline src/objectcontroller.cpp]: Would it be possible for you to add a unit test exercising both RGBWObjectValue's constructor and its ::toString()? This is all we can test, I believe.

Comment (fraxinas) [reply src/objectcontroller.cpp]: Hi Cyrille, i was going to write the tests when i ran into problems with the testbench.
what i tried was first installing `cppunit`, then in the `test/` subdir run `make testmain`.
When i do this, it does run the tests (also the ones i started implementing) but then throws hundreds of undefined references like this
```
testmain-ObjectControllerTest.o: In function `CppUnit::AutoRegisterSuite<ObjectControllerTest>::~AutoRegisterSuite()':
/usr/include/cppunit/extensions/AutoRegisterSuite.h:50: undefined reference to `CppUnit::TestFactoryRegistry::isValid()'
/usr/include/cppunit/extensions/AutoRegisterSuite.h:51: undefined reference to `CppUnit::TestFactoryRegistry::unregisterFactory(CppUnit::TestFactory*)'
testmain-ObjectControllerTest.o: In function `CppUnit::AdditionalMessage::~AdditionalMessage()':
```
i'm probably just running it wrong

Comment (2franix) [reply src/objectcontroller.cpp]: Hi Fraxinas,
Sorry for taking so long to reply. Do you have libcppunit-dev installed as well? I believe you miss the binary library to link against. If that does not help, please let me know which cppunit-related packages you have on your machine (apt search cppunit).

To run the tests, I usually issue "make check" in the test subfolder. I have never tried make testmain. But this is definitely not the cause of your problem anyway.

Let me know!

Comment (fraxinas) [reply src/objectcontroller.cpp]: Okay found the problem: my arch linux installed cppunit 1.14.0 while the configure.ac checks for specific version 1.9.6 i shall file a patch along with the finished unit tests

**Comment by fraxinas** [comment]
Okay i hope the tests are sufficient

**Comment by 2franix** [comment]
All set! Thanks again @fraxinas for this enhancement!

**Pull request merged**